### DISCUSSION TITLE
The Trash Can Button Isn't There?

### DISCUSSION THREAD

**Discussion by romifajari** [victornpb/undiscord#330] [1. Questions & Answers]
https://i.imgur.com/9u7Thlk.gif 

What I already tried: reinstalling the script, reinstalling tampermonkey, restarting firefox, disabled adblock

**Comment by Notonyyyy** [comment]
i also cant find it

**Comment by Quixilix** [comment]
Hello! I had the same issue for months, but I finally figured it out. You need to enable developer mode for your extesions, just follow their FaQ and you should be able to find it in Question(Q) 209! https://www.tampermonkey.net/faq.php#Q209
